homeassistant/components/hyperion/const.py

```diff
@@ -1,22 +1,29 @@
 """Constants for Hyperion integration."""
-DOMAIN = "hyperion"
-
-DEFAULT_NAME = "Hyperion"
-DEFAULT_ORIGIN = "Home Assistant"
-DEFAULT_PRIORITY = 128
+COLOR_BLACK = (0, 0, 0)
 
 CONF_AUTH_ID = "auth_id"
 CONF_CREATE_TOKEN = "create_token"
 CONF_INSTANCE = "instance"
 CONF_PRIORITY = "priority"
-
+CONF_MODE_OFF = "mode_off"
+CONF_MODE_OFF_LED_DEVICE = "led_device"
+CONF_MODE_OFF_SET_BLACK = "set_black"
 CONF_ROOT_CLIENT = "ROOT_CLIENT"
 CONF_ON_UNLOAD = "ON_UNLOAD"
 
-SIGNAL_INSTANCES_UPDATED = f"{DOMAIN}_instances_updated_signal." "{}"
-SIGNAL_INSTANCE_REMOVED = f"{DOMAIN}_instance_removed_signal." "{}"
+DEFAULT_MODE_OFF = CONF_MODE_OFF_LED_DEVICE
+DEFAULT_NAME = "Hyperion"
+DEFAULT_ORIGIN = "Home Assistant"
+DEFAULT_PRIORITY = 128
+
+DOMAIN = "hyperion"
```

Comment:
Please don't mix clean up with feature addition in the same PR. A clean up PR is usually super fast to review and merge. New features take longer time.